### PR TITLE
Delphi 12 support

### PR DESCRIPTION
**Pull request by rlebeau**

### CODE REVIEW

File: Lib/Core/IndyCore290.rc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+FILESUBTYPE 0x0L
+{
+ BLOCK "StringFileInfo"
+ {
+  BLOCK "000104E4"
+  {
+   VALUE "CompanyName", "Chad Z. Hower a.k.a Kudzu and the Indy Pit Crew\0"
+   VALUE "FileDescription", "Internet Direct (Indy) 10.6.2 - Core Run-Time Package\0"
+   VALUE "FileVersion", "10.6.2.0\0"
+   VALUE "InternalName", "IndyCore280\0"
```
Comment (RoLex) [inline Lib/Core/IndyCore290.rc:17]: Shouldn't it be 290 instead? And few more below.

Comment (rlebeau) [reply Lib/Core/IndyCore290.rc:17]: Yes, 290.

**Comment by rlebeau** [comment]
I've held off on merging this PR just yet, because I've been working recently on bringing Indy's [Package Generator](https://github.com/IndySockets/Indy/tree/master/Builder/Package%20Generator) up-to-date, including support for generating files for RADStudio 12.  That is almost ready, and then all merge any needed changes into this PR and merge everything into master.

**Comment by rlebeau** [comment]
I've merged in my own changes for Delphi 12, so I'm going to close this PR.  Resync your repo with the latest master code.

**Pull request closed without merging**